WBooks/SceneDelegate.swift

```diff
@@ -16,7 +16,8 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         guard let windowScene = (scene as? UIWindowScene) else { return }
         window = UIWindow(frame: windowScene.coordinateSpace.bounds)
         window?.windowScene = windowScene
-        window?.rootViewController = UINavigationController(rootViewController: LoginViewController())
+        let _loginViewModel = LoginViewModel()
```

Comment:
Also, we can create the view controller before the root view controller to be more clear and easy to change in the future if we need

Reply:
This is optional 😛

Reply:
Ok, thank you very much, I will keep it in mind to adjust it

Reply:
Done